picard/pluginmanager.py

```diff
@@ -445,3 +447,22 @@ def _plugins_json_loaded(self, response, reply, error, callback=None):
     # pylint: disable=no-self-use
     def enabled(self, name):
         return True
+
+    def check_update(self, parent=None):
+        if self.available_plugins is None:
+            self.query_available_plugins(partial(self._display_update, parent))
+        else:
+            self._display_update(parent)
+
+    def _display_update(self, parent):
+        if self.available_plugins is not None:
+            available_versions = {p.module_name: p.version for p in self.available_plugins}
+            for plugin in self.plugins:
+                if (plugin.module_name in available_versions and available_versions[plugin.module_name] > plugin.version):
+                    QMessageBox.information(
+                        parent,
+                        _("Picard Plugins Update"),
+                        _("There are updates available for your currently installed plugins."),
+                        QMessageBox.StandardButton.Ok, QMessageBox.StandardButton.Ok
+                    )
+                    break
```

Comment:
Since `_display_update()` is (and will be) only used in `check_update()`, I think we can revamp this a bit, see https://github.com/rdswift/picard/pull/17

Reply:
Merged.  Thanks.